### PR TITLE
[Docs] Gen AI

### PR DESCRIPTION
**Pull request by assouktim**

### CODE REVIEW

**Comment by assouktim** [comment]
@Benvii : Reviewed and validated internally by Agnes.

**Review by morgandiverrez — APPROVED** [review]

**Review by Benvii — CHANGES_REQUESTED** [review]
Didn't manage to read all of it but here are my first comments

File: docs/_fr/user/studio/gen-ai/features/gen-ai-feature-rag-exclusion.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+En complément du prompt, il permet de limiter les réponses du bot afin qu’il reste dans son périmètre d’action et ne crée pas de réponse trop inventive.
+
+Pour ajouter des sujets ou phrases que le bot ne doit pas traiter :
+1. Aller dans le menu **_Language Understanding / Inbox_**
+2. Sélectionner la phrase que vous souhaitez exclure
+3. Cliquer sur “Exclude from RAG handling”
+
+Sur l'écran **_Gen Ai / RAG Exclusions_**, vous verrez toutes les phrases/sujets qui ont été exclus des réponses du Bot.
+
+![RAG Exclusions](../../../../img/gen-ai/gen-ai-rag-excluded-2.png "Ecran des sujets exlus de l'IA")
```
[the remark this answers is not in the document]
Comment (assouktim) [reply docs/_fr/user/studio/gen-ai/features/gen-ai-feature-rag-exclusion.md:20]: comme vu ensemble, à rajouter après pour chaque feature.

[the remark this answers is not in the document]
Comment (assouktim) [reply docs/_fr/user/studio/gen-ai/features/gen-ai-feature-rag-exclusion.md:20]: Finalement, je l'ai ajouté ;)

**Comment by Benvii** [comment]
Todo : 

- [ ] `.md` links doesn't work
- [ ] table formatting to be done also

**Review by Benvii — APPROVED** [review]

**Pull request merged**